proto/dydxprotocol/accountplus/genesis.proto

```diff
@@ -3,10 +3,32 @@ package dydxprotocol.accountplus;
 
 import "gogoproto/gogo.proto";
 import "dydxprotocol/accountplus/accountplus.proto";
+import "dydxprotocol/accountplus/models.proto";
 
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/types";
 
+// AuthenticatorData represents a genesis exported account with Authenticators.
+// The address is used as the key, and the account authenticators are stored in
+// the authenticators field.
+message AuthenticatorData {
+  // address is an account address, one address can have many authenticators
+  string address = 1;
+
+  // authenticators are the account's authenticators, these can be multiple
+  // types including SignatureVerification, AllOfs, CosmWasmAuthenticators, etc
+  repeated AccountAuthenticator authenticators = 2
+      [ (gogoproto.nullable) = false ];
+}
+
 // Module genesis state
 message GenesisState {
   repeated AccountState accounts = 1 [ (gogoproto.nullable) = false ];
+
```

Comment:
Note that here I chose not to include the params ([link](https://github.com/osmosis-labs/osmosis/blob/dd225bc392ce72939003aa866f813a8c6d96a16f/proto/osmosis/smartaccount/v1beta1/params.proto)) for simplicity. 

not sure if we ever need to make this feature controllable by gov so omitting for now

Reply:
added!

osmosis actually also has a list of addresses that can bypass gov proposals ([link](https://github.com/osmosis-labs/osmosis/blob/362a01b08cd27d8e22e6c733a6f2c8c75b1d34df/proto/osmosis/smartaccount/v1beta1/params.proto#L22-L25)), omitted for now